release/planning.yaml

```diff
@@ -4,7 +4,7 @@ start: 2023-01-16  # The first week of the sprint
 messages:
   # Monday week2
   7:
-    text: "EUC Stack Release"
+    text: "@release-manager Please release EUC Stack"
   # Tuesday week2
   8:
-    text: "EUC Stack Release (reminder)"
+    text: "@release-manager Please release EUC Stack (reminder)"
```

Comment:
Is this an alias that exists?

Reply:
Perhaps we could create a custom group? It would need to be updated, but could be nice.

Reply:
Yep, this request has been sent to team-infra a lot of time ... without success to enable it quickly ... :joy: